### DISCUSSION TITLE
Bug in repositories view

### DISCUSSION THREAD

**Discussion by khamroevjs** [community/community#24075] [Apps, API and Webhooks]
<p>There is a excess repository which is not mine among my repositories. I’m not a contributor of this repository. I haven’t even forked it and don’t know how to delete it, because there is no a delete button.<br>
<img src="https://user-images.githubusercontent.com/55449509/181081812-223c6d81-f6a8-4424-bd07-4df6816a4524.png" alt="2021-08-11 18_02_37-GitHub" data-base62-sha1="kZoiHS2qNES0NgqTF3nVJQg1NoC" width="343" height="162"></p>
<p>Please help!<br>
Thanks in advance!</p>

**Comment by sudo-nautilus** [comment, marked as answer]
<p><a class="mention" href="/silentremix">@SilentRemix</a> welcome to the community</p>
<p>There’s no need to worry for that. It’s showing the repositories because you contributed to those (as an issue, pr, commit etc…). Lemme show mine to you<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/0/a0dbd2c050691879d923844535d6a6c9402f35e7.png" data-download-href="/uploads/short-url/mX1mAMfFgk4UPmr7aY234AP9PSv.png?dl=1" title="Screenshot_20210812_074934" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/78522210/181081814-66f807b0-a313-4439-88a1-6882268e4d72.png" alt="Screenshot_20210812_074934" data-base62-sha1="mX1mAMfFgk4UPmr7aY234AP9PSv" width="396" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/0/a0dbd2c050691879d923844535d6a6c9402f35e7_2_396x500.png, https://aws1.discourse-cdn.com/github/original/3X/a/0/a0dbd2c050691879d923844535d6a6c9402f35e7.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/a/0/a0dbd2c050691879d923844535d6a6c9402f35e7.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/0/a0dbd2c050691879d923844535d6a6c9402f35e7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_20210812_074934</span><span class="informations">528×665 31.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
See these are not entirely mine. But I contributed to those

**Comment by khamroevjs** [comment]
<p>Oh, now I got it. But is there any way to hide them?</p>

**Comment by sudo-nautilus** [comment]
<p>As far as I know, there’s no way to do that</p>

**Comment by khamroevjs** [comment]
<p>Ok, thank you. That was helpful</p>

**Comment by sudo-nautilus** [comment]
<p>You are welcome   …:wink:</p>
